src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp

```diff
@@ -119,6 +166,24 @@ class instance_stratholme : public InstanceMapScript
                         }
                         break;
                     }
+                    case NPC_PLAGUED_RAT:
+                    case NPC_PLAGUED_INSECT:
+                    case NPC_PLAGUED_MAGGOT:
+                    {
+                        auto el = PlaguedGatesGUIDs.find(who->GetGUID());
+                        if (el != PlaguedGatesGUIDs.end())
+                        {
+                            uint8 uiGate = el->second;
+                            PlaguedGatesGUIDs.erase(el);
+                        
```

Comment:
src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp:178:                        

Remove whitespace at the end of the lines above

Exited with code exit status 1